### PR TITLE
Fix broken markdown on I18n constant dev guive

### PR DESCRIPTION
**Pull request by jDramaix**
[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/77)

### CODE REVIEW

**Comment by jDramaix** [comment]
Someone could review?

File: src/main/markdown/doc/latest/DevGuideI18nConstants.md [hunk 1]
```diff
@@ -76,17 +76,17 @@ and must correspond to the return type of the method.  They provide a type-safe
...
-*   **`[@DefaultDoubleValue(double val)](/javadoc/latest/com/google/gwt/i18n/client/Constants.DefaultDoubleValue.html)`** Sets the default value for a method which returns a `double`.
-*   **`[@DefaultFloatValue(float val)](/javadoc/latest/com/google/gwt/i18n/client/Constants.DefaultFloatValue.html)`** Sets the default value for a method which returns a `float`.
-*   **`[@DefaultIntValue(int val)](/javadoc/latest/com/google/gwt/i18n/client/Constants.DefaultIntValue.html)`** Sets the default value for a method which returns a `int`.
-*   **`[@DefaultStringArrayValue({String str, ...})](/javadoc/latest/com/google/gwt/i18n/client/Constants.DefaultStringArrayValue.html)`** Sets the default value for a method which returns a `String` array.
-*   **`[@DefaultStringMapValue({String key, String value, ...})](/javadoc/latest/com/google/gwt/i18n/client/Constants.DefaultStringMapValue.html)`**
+*   **[@DefaultBooleanValue(boolean val)](/javadoc/latest/com/google/gwt/i18n/client/Constants.DefaultBooleanValue.html)** Sets the default value for a method which returns a `boolean`.
```
Comment (tbroyer) [inline src/main/markdown/doc/latest/DevGuideI18nConstants.md]: I think you can put the backticks inside the link label:

    **[`@DefaultBooleanValue(boolean val)`](…)**

Not sure about the rendering though ;-)

**Comment by tbroyer** [comment]
One comment/question; otherwise LGTM

**Comment by tbroyer** [comment]
LGTM

**Comment by manolo** [comment]
As @tbroyer said backticks work inside anchors, I tested as well with pegdown parser.

**Pull request merged**